velox/exec/HashBuild.cpp

```diff
@@ -94,9 +94,13 @@ HashBuild::HashBuild(
   }
 
   // Identify the non-key build side columns and make a decoder for each.
-  const auto numDependents = outputType->size() - numKeys;
-  dependentChannels_.reserve(numDependents);
-  decoders_.reserve(numDependents);
+  const int32_t numDependents = outputType->size() - numKeys;
```

Comment:
outputType name is confusing because it describe the input type for the join, not output type of the join. This is pre-existing, but any chance you could rename it inputType to help future readers?

Reply:
Yes, renamed.